int16 uav_id
geometry_msgs/Point uav_position
uint8 question
uint8 FREE_APPROACH_POINT=0
uint8 RESERVE_APPROACH_POINT=1
---
uint8 answer
uint8 WAIT=0
uint8 OK=1
geometry_msgs/Point approach_position
